public/less/take-a-stand.less

```diff
@@ -163,75 +192,104 @@
       }
     }
     .pol-pic-link {
-      font-size: 2rem;
+      // font-size: 2rem;
       font-weight: bold;
+      text-align: center;
       .text-font();
       > img {
         border-radius: 50%;
         border: 2px solid @blue;
         height: 18rem;
-        @media (min-width: @medium-bp) and (max-width: @large-bp) {
-          height: 15rem;
+        @media (max-width: @small-bp) {
+          width: 20rem;
+        }
+        @media (min-width: @small-bp) and (max-width: @medium-bp) {
+          height: 17rem;
+        }
+        @media (min-width: @medium-bp) {
+          height: 14rem;
         }
       }
     }
     .pol-name {
+      margin: auto;
+      text-align: center;
       margin-top: 1rem;
       min-height: 6rem; // uneven heights create irregular floating
       font-family: Oswald;
+      @media (max-width: @small-bp) {
+        font-size: 2rem;
+      }
+      @media (min-width: @small-bp) and (max-width: @medium-bp) {
+        font-size: 2rem;
+      }
+      @media (min-width: @medium-bp) {
+        font-size: 1.5rem;
+      }
     }
```

Comment:
This would be rewritten as 

font-size: 2rem
@media( min-width: @medium-bp){
font-size: 1.5rem
}